content/en/docs/concepts/configuration/overview.md

```diff
@@ -80,9 +80,7 @@ The [imagePullPolicy](/docs/concepts/containers/images/#updating-images) and the
 
 - `imagePullPolicy: Always`: the image is pulled every time the pod is started.
 
-- `imagePullPolicy` is omitted and either the image tag is `:latest` or it is omitted: `Always` is applied.
-
-- `imagePullPolicy` is omitted and the image tag is present but not `:latest`: `IfNotPresent` is applied.
+- `imagePullPolicy` is omitted and the image tag is present: `IfNotPresent` is applied.
```

Comment:
agree we should make this section easier to consume without resorting to first following the above link to #updating-images... but from my read, this edit presumes the behavior is wrong and should be changed.   

The active word "applied" is incorrect here, and I suspect that is a part of the confusion.  

"IfNotPresent" behavior is not really the default here.. instead what is happening is the default falls to an algorithm/table if not set.

Let's fix the docs without changing behavior.

Reply:
The default is ifNotPresent though, no? I guess I haven't worked on the code at all, but in all of my deployments, I've never seen K8s default to `never` or `always`.

Reply:
I would like to see the edits from @mikebrow before this is merged. I'm putting a hold on it because I want to make sure we get it right before its merged
/hold

Reply:
the default is ifnotpresent... but the default may be overriden depending on the image tag

Reply:
thus the default is a table of options

Reply:
default behavior is not equivalent to ifnotpresent behavior

this is an odd one